clinica/iotools/converters/oasis3_to_bids/oasis3_utils.py

```diff
@@ -248,9 +248,29 @@ def write_to_tsv(dataframe: DataFrame, buffer: Union[PathLike, BinaryIO]) -> Non
 
 
 def install_nifti(sourcedata_dir: PathLike, bids_filename: PathLike) -> None:
+    from pathlib import Path
+
     from fsspec.implementations.local import LocalFileSystem
 
     fs = LocalFileSystem(auto_mkdir=True)
+
+    path_end = Path(Path(Path(fs.ls(sourcedata_dir)[0]).stem).stem)
+    parent_dir = sourcedata_dir.parent
+    filename_stem = Path(bids_filename.stem).stem
+
+    for path in Path.joinpath(parent_dir, "BIDS").rglob(str(path_end) + "*"):
+        suffix = path.suffix
+        path_to_sidecar = Path.joinpath(parent_dir, "BIDS", path_end).with_suffix(
+            suffix
+        )
+        sidecar_filename = Path.joinpath(
+            bids_filename.parent, filename_stem
+        ).with_suffix(suffix)
+        source = fs.open(path_to_sidecar, mode="rb")
+        target = fs.open(sidecar_filename, mode="wb")
```

Comment:
let's keep `source_file` and `target_file` to stay consistent with the rest of `install_nifti`